apps/mobile/src/features/approver/components/outcomes-card.tsx

```diff
@@ -1,11 +1,33 @@
 import { BitcoinTokenBalance } from '@/features/balances/bitcoin/bitcoin-balance';
-import { useWalletTotalBitcoinBalance } from '@/queries/balance/bitcoin-balance.query';
+import { useBtcMarketDataQuery } from '@/queries/market-data/btc-market-data.query';
 import { t } from '@lingui/macro';
 
-import { AddressDisplayer, Avatar, Box, Flag, Text, UsersTwoIcon } from '@leather.io/ui/native';
+import { Money } from '@leather.io/models';
+import { AddressDisplayer, Avatar, Flag, Text, UsersTwoIcon } from '@leather.io/ui/native';
+import { baseCurrencyAmountInQuote, createMoney } from '@leather.io/utils';
+
+function BitcoinAddress({ address }: { address: string }) {
```

Comment:
Maybe this can be split into 3 component files? `BitcoinAddress`  + `BitcoinOutcome` + `OutcomeAddressesCard`?